src/components/MobileNavBar/NavBar.storybook.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { RouterDecorator } from '../../RouterDecorator';
+import { MobileNavBar } from '.';
+
+storiesOf('Mobile NavBar', module)
```

Comment:
With the `MobileNavBar` component grouped with the `NavBar`, you can nest the stories together to keep the root-level stories clean. I would move this `storiesOf` definition into the `src/components/NavBar/NavBar.storybook.tsx` file, and then you can nest it by naming it `'NavBar/Mobile'`, e.g.

```js
storiesOf('NavBar/Mobile', module)
  .add( /* whatever */)
```

More info on nesting stories in the documentation: https://storybook.js.org/basics/writing-stories/#nesting-stories